### PR TITLE
fix: update variable descriptions and metadata

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: examples/confidential_computing/main.tf [hunk 1]
```diff
@@ -65,7 +65,8 @@ resource "google_kms_crypto_key_iam_binding" "crypto_key" {
 }
 
 module "instance_template" {
-  source = "../../modules/instance_template"
+  source = "terraform-google-modules/vm/google//modules/instance_template"
```
Comment (iennae) [inline examples/confidential_computing/main.tf:68]: nit: // -> /

File: modules/mig/metadata.yaml [hunk 2]
```diff
@@ -80,61 +80,83 @@ spec:
         location: examples/compute_instance/tags
   interfaces:
     variables:
-      - name: autoscaler_name
-        description: Autoscaler name. When variable is empty, name will be derived from var.hostname.
+      - name: project_id
+        description: The GCP project ID
```
Comment (iennae) [inline modules/mig/metadata.yaml:84]: nit: The Google Cloud project ID

File: modules/mig/metadata.yaml [hunk 3]
```diff
@@ -80,61 +80,83 @@ spec:
...
-        varType: |-
-          list(object({
-              target            = number
-              predictive_method = string
-            }))
-        defaultValue: []
-      - name: autoscaling_enabled
-        description: Creates an autoscaler for the managed instance group
+      - name: region
+        description: The GCP region where the managed instance group resides.
```
Comment (iennae) [inline modules/mig/metadata.yaml:95]: nit: The Google Cloud region

File: examples/confidential_computing/main.tf (hunk 1 above)
Comment (apeabody) [reply examples/confidential_computing/main.tf:68]: Hi @iennae - I believe the `//` is required for sub-modules when using the registry?

https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

File: modules/mig/metadata.yaml (hunk 2 above)
Comment (apeabody) [reply modules/mig/metadata.yaml:84]: Sure - These are auto-generated, so we'll need to update the source template.

File: modules/mig/metadata.yaml (hunk 3 above)
Comment (apeabody) [reply modules/mig/metadata.yaml:95]: Sure - These are auto-generated, so we'll need to update the source template.

File: modules/mig/metadata.yaml (hunk 2 above)
Comment (apeabody) [reply modules/mig/metadata.yaml]: done - changed from chore to fix

File: modules/mig/metadata.yaml (hunk 3 above)
Comment (apeabody) [reply modules/mig/metadata.yaml]: Done

File: examples/confidential_computing/main.tf (hunk 1 above)
Comment (iennae) [reply examples/confidential_computing/main.tf]: oh woah! TIL! Thank you!!

**Review by iennae — APPROVED** [review]
LGTM.

**Review by g-awmalik — APPROVED** [review]

**Pull request merged**